rules/falco_rules.yaml

```diff
@@ -53,8 +53,8 @@
 - macro: never_true
   condition: (evt.num=0)
 
-- macro: always_true
-  condition: (evt.num>=0)
+# - macro: always_true
```

Comment:
I'd keep this and document how to use it. Same for `never_true`

Reply:
Can add comments re how to use, but how to keep it, CI failed, because we don't us it now?

Reply:
Our CI is too smart for us :D We can keep it commented I guess

Reply:
ok!